Campus-iOS/Crashlytics/CrashlyticsService.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CrashlyticsService.swift
+//  Campus-iOS
+//
+//  Created by Anton Wyrowski on 28.01.23.
+//
+
+import Foundation
+import FirebaseCrashlytics
+
+class CrashlyticsService {
```

Comment:
One minor thing: One could also put the call to the config method from Crashlytics into this service

Reply:
I thought of this too, but the config call is `FirebaseApp.configure()`. 
So the call is not only for crashlytics, but also for all other services of firebase and therefore perhaps wrong in the crashlytics service.

Reply:
Seems reasonable 👍